main.js

```diff
@@ -49,7 +49,7 @@ async function installEslintPackagesAsync () {
   const yarn = await getYarn()
 
   const versions = yarn.data.trees
-    .filter(p => p.name.match(/eslint/))
+    .filter(p => p.name.match(/eslint/) || p.name.match(/prettier/) || p.name.match(/babel/))
```

Comment:
I don't want to make this _too_ complicated, but this doesn't immediately strike me as well suited for all use cases.

I think we could make this a _little_ more complex (if it's worth the effort) and possibly match more closely to the "only install what's needed" intent of this action.

I _think_ it should be safe to have something along the lines of 

```js
if (yarn.data.trees.some(p => p.name.match(/eslint-prettier/)) {
  // install prettier
}
```

Similar thing for babel, where we could look at the eslint config to check and see if babel is _expected_ and then install it?

What do you think?

Reply:
I'm on-board with installing the minimum required dependencies. When it comes to _how_ to do it, I don't have any sense of direction on where to go. I took this approach because it was the most immediate thing I could see. I don't quite grok the intent of `getPeerDependencies`, but that seems like another candidate to get the names of peer dependencies, then install the versions from the yarn lock file?

Reply:
Yep, definitely a similar idea. 

_However_, I'm willing to mark this as good to go, merge it, and open an issue to improve that flow. I don't want to hold up actual linting